lib/alerts/alerts_api.rb

```diff
@@ -55,6 +55,63 @@ def get_single_trigger(trigger_id, full = false)
       trigger
     end
 
+    def bulk_load_triggers(hash)
```

Comment:
I am working on this PR to make it mergeable and I'd rename this method to `bulk_import_triggers` because `load` evokes the server -> client communication for me